invenio_search_ui/views.py

```diff
@@ -39,7 +38,5 @@
 
 @blueprint.route("/")
```

Comment:
imho we should go for `/search` or at least the blueprint should have default prefix `/search`